### ISSUE TITLE
CLI: panic on mcapp install

### ISSUE THREAD

**Issue by gitlawr** [rancher/rancher#18670]
**Cli version: v2.2.0-rc11**

**Steps to reproduce (least amount of steps as possible):**
1. Have a 2C4G single node K8S cluster with Rancher HA installed, and a GKE cluster provisioned
1. `$ rancher login -t xxx xxx`
1. `$ rancher mcapp install redis myredis`

**Result:**
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x13f137c]

goroutine 1 [running]:
github.com/rancher/cli/cmd.multiClusterAppTemplateInstall(0xc4200c4b00, 0xc42015cb00, 0xc4200c4b00)
	/go/src/github.com/rancher/cli/cmd/multiclusterapp.go:629 +0xd0c
github.com/rancher/cli/vendor/github.com/urfave/cli.HandleAction(0x1483360, 0x1580b00, 0xc4200c4b00, 0x0, 0xc420156300)
	/go/src/github.com/rancher/cli/vendor/github.com/urfave/cli/app.go:501 +0xd2
github.com/rancher/cli/vendor/github.com/urfave/cli.Command.Run(0x1562686, 0x7, 0x0, 0x0, 0x0, 0x0, 0x0, 0x156b460, 0x1b, 0x0, ...)
	/go/src/github.com/rancher/cli/vendor/github.com/urfave/cli/command.go:165 +0x4bb
github.com/rancher/cli/vendor/github.com/urfave/cli.(*App).RunAsSubcommand(0xc42013a380, 0xc4200c4840, 0x0, 0x0)
	/go/src/github.com/rancher/cli/vendor/github.com/urfave/cli/app.go:383 +0xad7
github.com/rancher/cli/vendor/github.com/urfave/cli.Command.startApp(0x1565cc8, 0x10, 0x0, 0x0, 0xc42007b8c0, 0x3, 0x3, 0x156f3aa, 0x22, 0x0, ...)
	/go/src/github.com/rancher/cli/vendor/github.com/urfave/cli/command.go:330 +0x944
github.com/rancher/cli/vendor/github.com/urfave/cli.Command.Run(0x1565cc8, 0x10, 0x0, 0x0, 0xc42007b8c0, 0x3, 0x3, 0x156f3aa, 0x22, 0x0, ...)
	/go/src/github.com/rancher/cli/vendor/github.com/urfave/cli/command.go:103 +0x8c9
github.com/rancher/cli/vendor/github.com/urfave/cli.(*App).Run(0xc42013a1c0, 0xc42010e300, 0x5, 0x8, 0x0, 0x0)
	/go/src/github.com/rancher/cli/vendor/github.com/urfave/cli/app.go:259 +0x740
main.mainErr(0xc42011ff70, 0x0)
	/go/src/github.com/rancher/cli/main.go:116 +0x1605
main.main()
	/go/src/github.com/rancher/cli/main.go:57 +0x22
```

**Other details that may be helpful:**
It does not happen in a standalone rancher. Inspecting the code, it is probably because the heavy loaded Rancher/K8S takes longer time to handle the conditions thus we get nil status when fetching the mcapp object after it is created.

**Comment by cjellick** [comment]
available in v2.2.0-rc13

**Comment by mrajashree** [comment]
Validated with cli v2.2.0-rc13. MCapp install did not panic
Command used: `./rancher mcapp install redis myredis`
Mcapp installation and corresponding app installation succeeded.
